cmd/podman/stop.go

```diff
@@ -73,21 +73,29 @@ func stopCmd(c *cliconfig.StopValues) error {
 		fmt.Println(err.Error())
 	}
 
+	if c.Flag("timeout").Changed && c.Flag("time").Changed {
+		return errors.New("the --timeout and --time flags are mutually exclusive")
+	}
```

Comment:
This upsets my delicate sensibilities: it would prevent a situation like:
```
alias podstop='podman stop --timeout=1'
...
$ podstop --time=5 mycontainer
```
(i.e. someone has an alias but doesn't remember exactly which spelling s/he used for --timeout).

Since the option var is the same in both instances, and the command-line processor will update them in the order seen: is this really necessary? It seems to work in practice.

Reply:
I saw wierd things happen when they were both specified (it was still using the 0 timeout), so my impulse is just disable using them together

Reply:
Weird how? I'm willing to hammer on it (in fact I have been) if you're willing to consider removing the restriction...

Reply:
@edsantiago How about I drop this to a warning log message saying only one of them will be used?

Reply:
Since the man page documents --timeout, --time, and -t as equivalent, what purpose would a warning message serve? If I were a user and saw such a message, I'd be very confused.